<ui version="4.0" >
 <class>DensityWindow</class>
 <widget class="QMainWindow" name="DensityWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Density Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionDensiteTex3D" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>DensiteTex3D</string>
   </property>
  </action>
  <action name="actionDensityON" >
   <property name="text" >
    <string>Afficher la Densité</string>
   </property>
  </action>
  <action name="actionDensityOFF" >
   <property name="text" >
    <string>Effacer la Densité</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
